lib/schema-evolution-manager/apply_util.rb

```diff
@@ -35,6 +36,24 @@ def apply!(dir)
       count
     end
 
+    def persist_pgpass(password)
+      Preconditions.check_not_blank(password, "password cannot be blank")
+      puts "Creating temp pgpass at #{@pgpass_file.path}"
+      FileUtils.chmod(0600, @pgpass_file.path)
+
+      FileUtils.rm_rf("#{ENV['HOME']}/.pgpass")
```

Comment:
This seems unintuitive for a user that you will nuke their pgpass file. There is a PGPASSWORD environment variable, which seems safer.